internal/command/test_test.go

```diff
@@ -301,8 +303,11 @@ func TestTest_Runs(t *testing.T) {
 		},
 		"ephemeral_input_with_error": {
 			expectedOut: []string{"Error message refers to ephemeral values", "1 passed, 1 failed."},
-			expectedErr: []string{"Test assertion failed", "has an ephemeral value"},
-			code:        1,
+			expectedErr: []string{"Test assertion failed", "has an ephemeral value",
+				`│ ~diff: 
+    | - (ephemeral value)
```

Comment:
I'm contemplating if we would rather show the real underlying value for ephemeral and sensitive values, given that it's just in test outputs

Reply:
Opted not to show marked values. Instead, we will expand the complex objects, and that provides better information to the user about the attributes that are marked. They may then decide to unmark them (e.g with `nonsensitive()`)